en_US/message-queue/message-queue-quick-start.md

```diff
@@ -0,0 +1,187 @@
+| `Least Inflight Subscriber` | Prefers subscribers with fewer in-flight (unacknowledged) messages. | Load balancing across uneven consumers |
+| `Round Robin`               | Delivers messages alternately to each subscriber in strict rotation. | Fair distribution regardless of speed  |
+| `Random` (default)          | Sends each message to a randomly chosen subscriber.          | Unpredictable or demo scenarios        |
+
+You can verify these behaviors by watching how messages are delivered to `worker-a` and `worker-b`.
+
+### Modify the Dispatch Strategy
+
+You can change the strategy on the fly:
+
+- Go to **Message Queue** in Dashboard.
+- Click **Edit** next to your queue.
+- Select a new **Dispatch Strategy** and save.
+
+After switching, repeat the message publishing test and observe the difference in distribution patterns between the subscribers.
+
+## Test Last-Value Semantics
+
+This section demonstrates how to enable **Last-Value Semantics**, which ensures only the most recent message per key is retained in the queue, ideal for use cases such as device configuration updates.
+
+### Step 1: Delete the Existing Queue
+
+1. Navigate to **Message Queue** in the EMQX Dashboard.
+2. Locate the queue with the topic filter `demo/topic`.
+3. Click **Delete** in the **Actions** column.
+4. Confirm deletion in the prompt.
+
+This removes the previous queue and its stored messages.
+
+### Step 2: Create a Queue with Last-Value Semantics
+
+1. On the **Message Queue** page, click **Create**.
+2. In the **Create Message Queue** dialog, configure the settings:
+   - **Topic Filter**: `device/config`
+   - **Dispatch Strategy**: `Random` (or your choice)
+   - **Data Retention Period**: `1` day
+   - **Last Value Semantics**: Toggle on
+   - **Queue Key Expression**: `message.key` (or any field name you will use as key)
+3. Click **Create**.
+
+The “Queue Key Expression” defines where to extract the key from the message’s metadata. Set to `message.key` means EMQX will extract the queue key from the client ID of the message publisher.
+
+### Step 3: Publish Messages
+
+1. Open MQTTX and select or create a client (e.g., `publisher`).
+
+2. Connect to EMQX (`mqtt://localhost:1883`).
+
+3. Publish messages to `device/config`:
+
+   Example:
+
+   | Field       | Value               |
+   | ----------- | ------------------- |
+   | **Topic**   | `device/config`     |
+   | **QoS**     | 1                   |
+   | **Payload** | `{"ssid": "wifi1"}` |
+
+4. Publish another message with updated content using the same client (same client ID):
+
+   ```json
+   Payload: {"ssid": "wifi2"}
+   ```
+
+Since the **Queue Key Expression** is set to `message.key`, EMQX will automatically extract the client ID from each message and use it as the queue key. Messages from the same client will overwrite previous unconsumed messages in the queue.
```

Comment:
```suggestion
Since the **Queue Key Expression** is set to `message.from`, EMQX will automatically extract the client ID from each message and use it as the queue key. Messages from the same client will overwrite previous unconsumed messages in the queue.
```